client/utils/planning.tsx

```diff
@@ -1057,15 +1058,26 @@ function createCoverageFromNewsItem(
         (ctype) => get(ctype, 'content item type') === addNewsItemToPlanning.type
     );
 
+    const contentTypeQcode = (contentType?.qcode ?? PLANNING.G2_CONTENT_TYPE.TEXT) as ICoverageType;
+
     newCoverage.planning = {
         ...newCoverage.planning,
-        g2_content_type: get(contentType, 'qcode', PLANNING.G2_CONTENT_TYPE.TEXT),
+        g2_content_type: contentTypeQcode,
         slugline: get(addNewsItemToPlanning, 'slugline', ''),
         ednote: get(addNewsItemToPlanning, 'ednote', ''),
         scheduled: moment().add(1, 'hour')
             .startOf('hour'),
     };
 
+    const allCoverageProfiles = coverageProfiles(planningApi.redux.store.getState());
```

Comment:
I think we need to stop doing this. It tighly couples the function to the external redux store making it much harder to write any tests. I would suggest to pass the computed values (`allCoverageProfiles`) as another parameter to the function. Ideally, you should use the `reselect` selectors we already have to grab this values and then pass them down. With that, it would be fairly simple to write a test and cover this change :)